### PR TITLE
Add Teku and Web3Signer TLS configuration information

### PR DESCRIPTION
**Pull request by bgravenorst**
## Pull Request Description
Add information to configure Teku <> Web3Signer TLS.

Dev PR: https://github.com/ConsenSys/teku/pull/3250

### CODE REVIEW

File: docs/HowTo/Configure/Configure-TLS.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+**Web3Signer prerequisites**:
+
+* [Web3Signer's password-protected PKCS12 keystore and password file].
+* [Web3Signer known clients file].
+
+**Teku prerequisites**:
+
+* [Teku's password-protected PKCS12 or JKS keystore and password file].
+* [Password-protected truststore and password file].
```
Comment (usmansaleem) [inline docs/HowTo/Configure/Configure-TLS.md:24]: note: the truststore is also a PKCS12 or JKS format keystore. It is only storing Web3Signer's self-signed certificate or CA certificate which signs Web3Signer's certificate.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -1468,6 +1527,62 @@ using Teku to sign blocks and attestations always uses its built-in slashing pro
...
+    TEKU_VALIDATORS_EXTERNAL_TRUSTSTORE=websigner_truststore.p12
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    validators-external-signer-truststore: "websigner_truststore.p12"
+    ```
+
+PKCS12 or JKS keystore used to trust external signer's self-signed certificate
```
Comment (usmansaleem) [inline docs/Reference/CLI/CLI-Syntax.md:1556]: suggestion similar to: `PKCS12 or JKS keystore used to trust external signer's self-signed certificate or CA certificate which signs external signer's certificate`

File: docs/Tutorials/Configure-External-Signer-TLS.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+Configure TLS communication with an external signer such as [Web3Signer] which accepts
+connections from clients that use trusted CA certificates or self-signed certificates.
+
+This tutorial configures TLS between Teku and Web3Signer, and use the
+[`keytool`](https://docs.oracle.com/en/java/javase/12/tools/keytool.html) utility to generate
+keystores and the truststore that contain self-signed certificates.
+
+!!! info
+
+    `keytool` is available with the JDK installation, you can also use OpenSSL.
```
Comment (usmansaleem) [inline docs/Tutorials/Configure-External-Signer-TLS.md:16]: keytool is available with both JRE/JDK installation

File: docs/Tutorials/Configure-External-Signer-TLS.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+
+1. Generate the Web3Signer keystore.
+
+    ```bash
+    keytool -genkeypair -keystore web3signer_keystore.p12 -storetype PKCS12 -storepass changeit -alias web3signer -keyalg RSA -keysize 2048 -validity 109500 -dname "CN=localhost, OU=PegaSys, O=ConsenSys, L=Brisbane, ST=QLD, C=AU" -ext san=dns:localhost,ip:127.0.0.1
+    ```
+
+    !!! info
+
+        `CN` is generally the fully qualified name of Web3Server, you can use `-ext san` to add additional
```
Comment (usmansaleem) [inline docs/Tutorials/Configure-External-Signer-TLS.md:51]: `CN` (common name) is generally the fully qualified host name of Web3Server

**Review by usmansaleem — APPROVED** [review]
LGTM - minor comments/suggestions that can be incorporated.

File: docs/Tutorials/Configure-External-Signer-TLS.md (hunk 3 above)
Comment (bgravenorst) [reply docs/Tutorials/Configure-External-Signer-TLS.md:16]: Done

File: docs/Reference/CLI/CLI-Syntax.md (hunk 2 above)
Comment (bgravenorst) [reply docs/Reference/CLI/CLI-Syntax.md:1556]: Done

File: docs/Tutorials/Configure-External-Signer-TLS.md (hunk 4 above)
Comment (bgravenorst) [reply docs/Tutorials/Configure-External-Signer-TLS.md:51]: Done

**Pull request merged**